core/com.b2international.snowowl.datastore/src/com/b2international/snowowl/datastore/CodeSystemEntry.java

```diff
@@ -340,4 +343,15 @@ public boolean equals(final Object obj) {
 			return false;
 		return true;
 	}
-}
\ No newline at end of file
+	
+	/**
+	 * Returns all code system short name dependencies and itself.
+	 */
+	@JsonIgnore
+	public List<String> getAffectedCodeSystems() {
```

Comment:
Rename to `getDependencies()`.

Reply:
Or rename it to `getDependenciesAndSelf()` and the method should return a `java.util.SortedSet<E>`.